app/Http/Controllers/Admin/UsersController.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace App\Http\Controllers\admin;
+
+use Illuminate\Http\Request;
+
+use App\Http\Requests;
+use App\http\Requests\UsersRequest;
+use App\Http\Controllers\Controller;
+use App\User;
+use Datatables;
+use App\Helpers\GeneralHelper;
+use Flash;
+
+class UsersController extends Controller
+{
+    /**
+     * Index users
+     *
+     * @return view index users
+     */
+    public function index()
+    {
+        return view('admin.users.index');
+    }
+
+    /**
+     * Function GET create Users
+     *
+     * @return view create.
+     */
+    public function getCreate()
```

Comment:
Follow Laravel convention https://laravel.com/docs/5.2/controllers#restful-resource-controllers
 The function name should be `create` not `getCreate`. You should use resource controller instead of define route one by one